File: getAstaAPIToken.js
import fs from 'fs';

function getTokenFromFile() {
    const filePath = ".mysecret.txt" // Is in the .gitignore
    try {
        const token = fs.readFileSync(filePath, 'utf8').trim();
        return token;
    } catch (error) {
        console.error(`Error reading token from file '${filePath}':`, error.message);
        return null;
    }
}

export default getTokenFromFile;

File: getAstaAPIToken.py
def get_token_from_file():
    file_path = ".mysecret.txt"  # Is in the .gitignore
    try:
        with open(file_path, 'r') as file:
            token = file.read().strip()
            return token
    except Exception as error:
        print(f"Error reading token from file '{file_path}': {error}")
        return None
